sdk/python/jobs/automl-standalone-jobs/jsonl-conversion/coco_jsonl_converter.py

```diff
@@ -0,0 +1,176 @@
+from base_jsonl_converter import JSONLConverter
+from azureml.automl.dnn.vision.object_detection.common import masktools
```

Comment:
Why do we need to install azureml-automl-dnn-vision and import this here. Previously, we used to copy a portion of code that is required to this script and use it.
Installing azureml-automl-dnn-vision with all its dependencies might be a little heavy on the client side.

Reply:
Ah okay, I thought that it was redundant code because we already had the code in automl-dnn-vision package. I'll copy the necessary code in to avoid install + import

Reply:
Verified refactor by comparing jsonl files before and after, they are the same